### ISSUE TITLE
Строка поиска динамического списка

### ISSUE THREAD

**Issue by PRO100MIXA** [SeiOkami/OneS#24]
Добавить возможность переопределять поля по которым выполняется поиск в динамических списках, указав только те поля по которым стоит совершать поиск.

Найти что-нибудь даже в простых динамических списках (основная таблица документа и регистр сведений со статусами документа в пару млн строк) может быть слишком долго + по словам поддержки полнотекстовый поиск искать по вхождению строки не умеет. А при отсутствии результата полнотекстового поиска выполняется поиск по СУБД с очень большими запросами.

Если почитать статьи связанные с тормозами динамических списков - можно увидеть советы: 
Использовать другой софт для индексации данных
Делать регистры сведений с сотнями миллионов комбинаций искомых слов
Отключение строки поиска (или ищи по Alt+F)

**Comment by PawloAkulow** [comment]
да, и иметь возможность искать по колонкам которые не видно в данный момент

**Comment by SeiOkami** [comment]
Опубликовано на канале: https://t.me/UnionOneS/58
Отправлено в чат: https://t.me/e1c_community/47083
Отправлено боту 1С в 09:12
